Lib/collections/__init__.py

```diff
@@ -908,6 +909,35 @@ def __and__(self, other):
                 result[elem] = newcount
         return result
 
+    def __xor__(self, other):
+        '''Symmetric difference. Absolute value of count differences.
+
+        The symmetric difference p ^ q is equivalent to:
+
+            (p - q) | (q - p).
+
+        For each element, symmetric difference gives the same result as:
+
+            max(p[elem], q[elem]) - min(p[elem], q[elem])
+
+        >>> Counter(a=5, b=3, c=2, d=2) ^ Counter(a=1, b=3, c=5, e=1)
+        Counter({'a': 4, 'c': 3, 'd': 2, 'e': 1})
+
+        '''
+        if not isinstance(other, Counter):
+            return NotImplemented
+        result = Counter()
+        for elem, count in self.items():
+            newcount = abs(count - other[elem])
+            if newcount:
+                result[elem] = newcount
+        for elem, count in other.items():
+            if elem not in self:
+                newcount = abs(count)
+                if newcount:
+                    result[elem] = newcount
```

Comment:
```suggestion
            if elem not in self and count:
                result[elem] = abs(count)
```

I think this would be equivalent, unless you want to support "broken" counts for which `bool(count) != bool(abs(count))` (or maybe you wrote it like that to match the previous loop)

Reply:
I was just matching the previous loop.  Keeping the parallel construction was easier for me to reason about.